core/vm/logger.go

```diff
@@ -106,7 +106,7 @@ func (s *StructLog) ErrorString() string {
 // if you need to retain them beyond the current call.
 type Tracer interface {
 	CaptureStart(from common.Address, to common.Address, create bool, input []byte, gas uint64, value *big.Int) error
-	CaptureState(env *EVM, pc uint64, op OpCode, gas, cost uint64, memory *Memory, stack *Stack, rStack *ReturnStack, rData []byte, contract *Contract, depth int, err error) error
+	CaptureState(env *EVM, pc uint64, op OpCode, gas, availableGas, cost uint64, memory *Memory, stack *Stack, rStack *ReturnStack, rData []byte, contract *Contract, depth int, err error) error
```

Comment:
@ziogaschr I think you mentioned yesterday that this parameter may be removable since this value is equivalent to `env.callGasTemp`. If so, is it worth doing?

Reply:
At this point removed the extra `availableGasCopy` and use the `in.evm.callGasTemp`. This one can be removed if I make it public as `in.evm.CallGasTemp`. What do you think about it?

Reply:
I think it's OK to make it public because that follows the pattern that we've already "allowed" with the introduction of `EVM.CallErrorTemp`, and IMO publicizing a field there is less "expensive" than adding a parameter to a shared interface.